task_01/ex5.py

```diff
@@ -0,0 +1,12 @@
+"""Functions."""
+
+
+def greet(name):
+    """Print a greeting to an argument."""
+    print('Hello,', name)
+
+
+greet('Jack')
+greet('Jill')
+greet('Bob')
+greet(input())
```

Comment:
It would be nice to write smth like "Please enter your name" before this input because for me as user it would be unclear what's happening and what should I do